### PR TITLE
dev commit

### PR DESCRIPTION
**Pull request by shaifpatel**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -4,9 +4,11 @@ const userRouter = require('./routes/userRouter')
...
 const port = 3000
 const db =require('./model/index')
+
 require('./model/index')
 
 
+
 app.use(express.json())
 
 app.use('/users',userRouter)
```
Comment (taist24) [inline app.js:14]: Use a formatter and linter such as ESLint and Prettier.

Refer: https://dev.to/devdammak/setting-up-eslint-in-your-javascript-project-with-vs-code-2amf

File: controller/hashtagcontroller.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ const createhashtags = async (req, resp) => {
     resp.status(201).json({ data: data, message: "hashtag create successfully created" })
 }
```
Comment (taist24) [inline controller/hashtagcontroller.js:13]: Use capital case for file names: Eg: HashTagController.js

File: controller/usercontroller.js [hunk 3]
```diff
@@ -6,7 +6,6 @@ const jwt_key = "jwt"
 
 
 
-
 ////////////   signup     ////////
 const signUp = async (req, resp) => {
     const { name, email, password } = req.body
```
Comment (taist24) [inline controller/usercontroller.js:11]: Validate incoming request body using joi package.

https://www.npmjs.com/package/joi

File: routes/hashtagRouter.js [hunk 4]
```diff
@@ -3,9 +3,8 @@ const { createhashtags, getHashtag, getallhashtag } = require('../controller/has
 const auth = require('../middleware/auth')
 const hashtagRouter = express.Router()
 
-hashtagRouter.post('/',auth,createhashtags)
-
-hashtagRouter.get('/',auth,getHashtag)
-hashtagRouter.get('/allhastag',getallhashtag)
+hashtagRouter.post('/', auth, createhashtags)
+hashtagRouter.get('/', auth, getHashtag)
+hashtagRouter.get('/allhastag', getallhashtag)
```
Comment (taist24) [inline routes/hashtagRouter.js:8]: Better route name would be:

/hashtag/all

File: routes/userRouter.js [hunk 5]
```diff
@@ -2,10 +2,10 @@ const express = require('express')
...
 
-userRouter.post('/signup',signUp)
-userRouter.post('/login',login)
+userRouter.post('/signup', signUp)
+userRouter.post('/login', login)
 
-userRouter.post('/login',(req,resp)=>{
+userRouter.post('/login', (req, resp) => {
     resp.send("login")
 })
```
Comment (taist24) [inline routes/userRouter.js:10]: Why two routes with same endpoint are defined?

File: middleware/auth.js [hunk 6]
```diff
@@ -11,8 +11,6 @@ const auth = async (req, resp, next) => {
             token = token.split(' ')[1]
             const data = jwt.verify(token, jwt_key)
             req.userId = data.id
```
Comment (taist24) [inline middleware/auth.js:13]: Use only camelCase convention. Don't use snake_case in js.

File: app.js (hunk 1 above)
Comment (shaifpatel) [reply app.js:14]: ok,

File: middleware/auth.js (hunk 6 above)
Comment (shaifpatel) [reply middleware/auth.js:13]: ok next time use camelCase

File: routes/userRouter.js (hunk 5 above)
Comment (shaifpatel) [reply routes/userRouter.js:10]: sorry , i forgget to remove one

File: controller/hashtagcontroller.js (hunk 2 above)
Comment (shaifpatel) [reply controller/hashtagcontroller.js:13]: ok

**Pull request merged**